screen2.0/src/app/search/ccredetails/tfintersection.tsx

```diff
@@ -80,14 +81,14 @@ export const TfIntersection: React.FC<{ assembly: string; coordinates: { chromos
   return (
     <>
       {loading || !data ? (
-        <Grid2 container spacing={3} sx={{ mt: "1rem", mb: "2rem" }}>
-          <Grid2 xs={12} lg={12}>
-            <CircularProgress />
+        <Grid2 container spacing={3} sx={{ mt: "0rem", mb: "2rem" }}>
+          <Grid2 xs={12} md={12} lg={12}>
```

Comment:
Redundant breakpoints, can just use xs={12} unless you want it to resize on different screen widths